test-tools/wamr-ide/VSCode-Extension/src/utilities/lldbUtilities.ts

```diff
@@ -67,18 +68,17 @@ export function isLLDBInstalled(context: vscode.ExtensionContext): boolean {
 
 export async function promptInstallLLDB(
     context: vscode.ExtensionContext
-): Promise<void> {
+): Promise<string> {
```

Comment:
I'd prefer not returning `string` from a function that returns well-defined subset of values. Looking at the usage of this function, my recommendation would be to return `SelectionOfPrompt` (and return `setUp` at the end of this function instead of `Status` object).

Reply:
good suggestion, updated